eZ/Publish/API/Repository/Tests/URLAliasServiceTest.php

```diff
@@ -1693,4 +1693,26 @@ private function insertBrokenUrlAliasTableFixtures(Connection $connection)
 
         return count($rows);
     }
+
+    public function testCreateUrlAliasWithRomanianReservedCharacters()
```

Comment:
Please change the test to test lookup after publishing. As discussed you could use `$this->createFolder` to simplify code.